### ISSUE TITLE
[2.0.0-beta.6] TypeError when submitting form with Email field

### ISSUE THREAD

**Issue by mtnorthrop** [verbb/formie#892]
**Description**
I get the following error when submitting a form with an Email field:
```
TypeError
Cannot assign craft\db\Query to property verbb\formie\events\ModifyEmailFieldUniqueQueryEvent::$query of type ?craft\elements\db\ElementQueryInterface
```

It looks like the problem is in `verbb\formie\fields\formfields\Email::validateUniqueEmail()`, on lines https://github.com/verbb/formie/blob/1482b90bad2957c505df476fa1255631d3a76406/src/fields/formfields/Email.php#L129-L140

The `$query` variable is an instance of `craft\db\Query`, but the type of the `$query` property in the event it's being passed to is set to `?ElementQueryInterface`, and `Query` doesn't implement this interface:

https://github.com/verbb/formie/blob/1482b90bad2957c505df476fa1255631d3a76406/src/events/ModifyEmailFieldUniqueQueryEvent.php#L15

**Additional info**
- Plugin version: 2.0.0-beta.6
- Craft version: 4.0.2
- PHP version: 8.0.13

**Comment by engram-design** [comment]
Completely missed this, but you're correct. Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.6",
  "...": "..."
}
```

Then run `composer update`.

**Comment by mtnorthrop** [comment]
Works great! Thanks so much for the quick fix!

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.7](https://github.com/verbb/formie/releases/tag/2.0.0-beta.7)
